packages/ppx_default/ppx_default.0.1.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Generate default values for your types"
+maintainer: ["ProgramingIsTheFuture"]
+authors: ["ProgramingIsTheFuture"]
```

Comment:
There is no problem with using a screen name as the author, but just so you know, most people use their real name in packages on opam-repository.